webpack.common.js

```diff
@@ -0,0 +1,44 @@
+const HtmlWebPackPlugin = require("html-webpack-plugin");
+
+module.exports = {
+    entry: './src/index.js',
+    module: {
+        rules: [
+            {
+                test: /\.(js|jsx)$/,
+                exclude: /node_modules/,
+                use: [{loader: 'babel-loader', options: {presets: ['@babel/preset-env', '@babel/react']}}]
+            },
+            {
+                test: /\.html$/,
```

Comment:
It's don't need to add `html-loader` here. You html template is parsed by `HtmlWebPackPlugin `.